### ISSUE TITLE
ClipboardCopy/CodeBlock - Additional action in example missing a tooltip

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11385]
**Describe the enhancement or change**
In the [inline compact with additional action clipboard copy example](https://www.patternfly.org/components/clipboard-copy#inline-compact-with-additional-action) and the [codeblock examples](https://www.patternfly.org/components/code-block), the second action button should have a tooltip since it may not be an easily recognizable aciton in the context of a clipboard copy.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

**Comment by Mash707** [comment]
Hi @thatblindgeye would like to work on this, are we talking about something like this?

![Image](https://github.com/user-attachments/assets/519f8d0f-6969-4a1b-adc1-a4106b76fc5b)
It is present under the [design guidelines](https://www.patternfly.org/components/code-block/design-guidelines) .
Should we add this tooltip in [codeblock examples](https://www.patternfly.org/components/code-block) also or is it self-explanatory?

**Comment by thatblindgeye** [comment]
Yep, for these additional actions the default tooltip text could be "Run in web terminal", and upon clicking the button the tooltip content could update to what is shownin that screenshot (similar to the copy button defaulting to "Copy to clipboard" and updating to "Copied to clipboard" upon clicking).

We'd want the linked ClipboardCopy example updated as well as any of the Codeblock examples.
